src/couch/src/couch_httpd_auth.erl

```diff
@@ -371,9 +371,18 @@ cookie_auth_cookie(Req, User, Secret, TimeStamp) ->
     mochiweb_cookies:cookie(
         "AuthSession",
         couch_util:encodeBase64Url(SessionData ++ ":" ++ ?b2l(Hash)),
-        [{path, "/"}] ++ cookie_scheme(Req) ++ max_age() ++ cookie_domain() ++ same_site()
+        cookie_attributes(Req)
     ).
 
+clear_auth_cookie(Req) ->
+    mochiweb_cookies:cookie(
+      "AuthSession", "", cookie_attributes(Req)
+     ).
+
+cookie_attributes(Req) ->
+    Attributes = [path(), http_only(), max_age(), cookie_scheme(Req), max_age(), cookie_domain(), same_site()],
```

Comment:
`max_age()` specified twice. `mochiweb_cookies:cookie/1` "uniquifiies" so I think that explains why we didn't see it in the headers.

Reply:
oops. fixed.